### PR TITLE
More fixes after merge

### PR DESCRIPTION
**Pull request by christianbaroni**

### CODE REVIEW

File: src/components/savings/SavingsSheetHeader.js [hunk 1]
```diff
@@ -18,15 +17,14 @@ const SavingsSheetHeader = ({ balance, lifetimeAccruedInterest }) => (
...
-      <Icon name="plusCircled" color={colors.green} />
       <Text
         color={colors.green}
         letterSpacing="roundedTight"
         size="large"
         lineHeight="loose"
         weight="semibold"
       >
-        {lifetimeAccruedInterest}
+        􀁍 {lifetimeAccruedInterest}
```
Comment (brunobar79) [inline src/components/savings/SavingsSheetHeader.js]: (Just making sure that you want this character)

File: src/screens/SavingsSheet.js [hunk 2]
```diff
@@ -61,8 +61,7 @@ const SavingsSheet = () => {
           <RowWithMargins css={padding(24, 15)} margin={15}>
             <SheetActionButton
               color={colors.dark}
-              icon="minusCircled"
-              label="Withdraw"
+              label="􀁏 Withdraw"
```
Comment (brunobar79) [inline src/screens/SavingsSheet.js]: same here

File: src/screens/SavingsSheet.js [hunk 3]
```diff
@@ -71,29 +70,39 @@ const SavingsSheet = () => {
...
+              shadows={[
+                [0, 7, 21, colors.dark, 0.25],
+                [0, 3.5, 10.5, colors.dark, 0.35],
+              ]}
             />
             <SheetActionButton
               color={colors.swapPurple}
-              icon="plusCircled"
-              label="Deposit"
+              label="􀁍 Deposit"
```
Comment (brunobar79) [inline src/screens/SavingsSheet.js]: and here

**Comment by christianbaroni** [comment]
> LGTM! just making sure those non standards chars are something you want.

Ah yeah, actually wanted to check with you and @mikedemarais about that. So the font we're using (SF Pro Rounded) has an icon library built into it, and often in my designs I'll just use those icons in the text layer. Doing the same in implementation just makes things more consistent.

Only worry was incompatibility with Android, but I'm in favor of using the same font on Android unless there's a compelling reason not to, so it should work fine there too.

Thoughts?

**Comment by brunobar79** [comment]
As long as the font has the character, we should be good!

**Comment by mikedemarais** [comment]
@christianbaroni does it work? like do those characters render properly in app?

**Comment by christianbaroni** [comment]
Yeah they do, they just don't render if displayed in a font that doesn't have the icons.

**Pull request closed without merging**